lib/datadog/tracing/contrib/sinatra/env.rb

```diff
@@ -1,6 +1,7 @@
 require 'time'
 
 require_relative '../../metadata/ext'
+require_relative '../rack/header_collection'
```

Comment:
It looks like we are not using `header_collection` on this file 😄

Reply:
Nice catch, this is an artifact from early development. 👍